server/lib/template/helpers/finance.js

```diff
@@ -36,7 +36,13 @@ function currency(value = 0, currencyId, digit) {
   } else {
     // usage with custom digits for precision
     const format = JSON.parse(JSON.stringify(fmt));
-    format.precision = digit;
+
+    /*
+      * lomamech : there had been a problem with the display of digit after the decimal
+      * point after investigation, I realized that the value "digit" was undefined
+    */
+    // format.precision = digit;
```

Comment:
Can we just set this to be

```js
if (digit !== undefined) { format.precision = digit; }
```

I think it was originally there for a purpose.